### PR TITLE
Tiny fixes

### PR DESCRIPTION
**Pull request by swansontec**
### CHANGELOG

Does this branch warrant an entry to the CHANGELOG?

- [ ] Yes
- [x] No

### Dependencies

 none

### Requirements

If you have made **any** visual changes to the GUI. Make sure you have:

- [x] Tested on iOS device
- [ ] Tested on Android device
- [ ] Tested on small-screen device (iPod Touch)
- [ ] Tested on large-screen device (tablet)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207780186591855

### CODE REVIEW

File: src/components/scenes/CoinRankingScene.tsx [hunk 1]
```diff
@@ -144,6 +146,12 @@ const CoinRankingComponent = (props: Props) => {
     }
   }, [])
 
+  React.useEffect(() => {
+    return navigation.addListener('focus', () => {
+      dispatch(checkEnabledExchanges())
+    })
+  }, [dispatch, navigation])
```
Comment (samholmes) [inline src/components/scenes/CoinRankingScene.tsx:153]: Isn't there a `useFocusEffect` hook from react-navigation you could use?

File: src/components/scenes/SwapCreateScene.tsx [hunk 2]
```diff
@@ -107,6 +108,16 @@ export const SwapCreateScene = (props: Props) => {
...
 
+  //
+  // Effects
+  //
+
+  React.useEffect(() => {
+    return navigation.addListener('focus', () => {
+      dispatch(checkEnabledExchanges())
+    })
+  }, [dispatch, navigation])
```
Comment (samholmes) [inline src/components/scenes/SwapCreateScene.tsx:119]: Same: `useFocusEffect` suggestion here.

File: src/components/scenes/SettingsScene.tsx [hunk 3]
```diff
@@ -298,6 +299,15 @@ export const SettingsScene = (props: Props) => {
...
   }, [context, supportsTouchId])
 
+  // Show a modal if we have a pending OTP resent when we enter the scene:
+  React.useEffect(() => {
+    return navigation.addListener('focus', () => {
+      if (account.otpResetDate != null) {
+        showReEnableOtpModal(account).catch(error => showError(error))
+      }
+    })
+  }, [account, navigation])
```
Comment (samholmes) [inline src/components/scenes/SettingsScene.tsx:309]: `useFocusEffect`?

**Review by samholmes — CHANGES_REQUESTED** [review]

File: src/components/scenes/CoinRankingScene.tsx (hunk 1 above)
Comment (swansontec) [reply src/components/scenes/CoinRankingScene.tsx:153]: No, that's not appropriate for this use-case. There is a lot of extra logic in that hook that we don't need.

**Review by samholmes — APPROVED** [review]

**Pull request merged**